main/src/ca/uwaterloo/flix/api/lsp/provider/ShowAstProvider.scala

```diff
@@ -16,17 +16,15 @@
 package ca.uwaterloo.flix.api.lsp.provider
 
 import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.api.Flix.IrFileExtension
 import ca.uwaterloo.flix.api.lsp.Index
 import ca.uwaterloo.flix.language.ast.TypedAst.Root
-import ca.uwaterloo.flix.language.dbg.PrettyPrinter
-import ca.uwaterloo.flix.util.Formatter
+import ca.uwaterloo.flix.language.dbg.AstPrinter.formatLiftedAst
```

Comment:
We usually dont import methods... Although that would be the Flix way :)

Reply:
I wish you didn't have to import functions in flix 😄